man/is.sorted.Rd

```diff
@@ -0,0 +1,50 @@
+\name{is.sorted}
+\alias{is.sorted}
+\title{ Checks if input is sorted }
+\description{
+  Checks if input is sorted.
+}
+\usage{
+  is.sorted(x, by=seq_along(x), retOrd=FALSE)
+}
+\arguments{
+  \item{x}{ data.table type object or atomic vector. }
+  \item{by}{ data.table columns used to check if \code{x} is sorted by those columns. }
```

Comment:
```by = seq_along(x)``` implies an integer argument. Since a character vector is supported, we may want to point it out.